### PR TITLE
#168367131 Add a sign up with posgres endpoint

### PR DESCRIPTION
**Pull request by Mbonigabay**
#### What does this PR do?

- Build a signup endpoint that works with Postgres

#### Description of Task to be completed?

- Setup my application to work with postgres

- Add a signup endpoint
#### How should this be manually tested?

- By checking the endpoint using postman

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [https://www.pivotaltracker.com/story/show/168367131](https://www.pivotaltracker.com/story/show/168367131)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -64,6 +64,18 @@
     "@babel/traverse" "^7.4.4"
```
Comment (Kasulejoseph) [inline yarn.lock]: Ignore the yarn lock file

File: app/controller/userController.js [hunk 2]
```diff
@@ -6,248 +6,73 @@ import jwt from 'jsonwebtoken';
...
-
+            return res.status(400).json({
+              status: 400,
+              message: 'server error please try again later',
+            });
+          }
+        } finally {
+          client.release()
+        }
+      })().catch(e => console.log(e.stack))
```
Comment (Kasulejoseph) [inline app/controller/userController.js]: You need to handle this error and not just console logging it..

File: app/route.rest [hunk 3]
```diff
@@ -1,18 +1,23 @@
+# Get all users
```
Comment (Kasulejoseph) [inline app/route.rest]: Any reason you're not using POSTMAN? If this is development purposes or debuggering then you need to ignore this file or remove it

Comment (Mbonigabay) [reply app/route.rest]: It is in my .gitignore but everytime it is pushed too

File: yarn.lock (hunk 1 above)
Comment (Mbonigabay) [reply yarn.lock]: I deleted it on the pull request #16

Comment (Kasulejoseph) [reply yarn.lock]: Can you delete it from here as well

Comment (Mbonigabay) [reply yarn.lock]: `i have done it`

File: app/route.rest (hunk 3 above)
Comment (Kasulejoseph) [reply app/route.rest]: You need to first delete it from remote, I gave you a resource

**Pull request merged**